### ISSUE TITLE
2.5.4 Motion Actuation and camera access (for QR codes etc, not actual "gesturing towards the device")

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag#1404]
I've seen questions bubbling up a few times (most recently today https://webaim.org/discussion/mail_message?id=45215) about whether or not functionality like QR code reading fails 2.5.4 because "users have to move their device to point the camera at the QR code". 

my take is that no, this does not fail, because the "motion" - having to potentially move the device to point the camera to the QR code - is ancilliary ... the motion itself is not what triggers any functionality. the app/site is not reacting to the motion per se. originally, from my memories on the MATF at the time, the intent was essentially a specific scenario of a more general "reacting to sensors, like gyros, near-field sensors, accelerometers, etc.", but pared down to just cover motion.

the understanding document does, confusingly, mention cameras (in relation to users gesturing towards the device), but perhaps it needs to add an extra note about things like QR codes?

**Comment by patrickhlauke** [comment]
ah, i see we've had this here already earlier in the year, but with no hard conclusion https://github.com/w3c/wcag/issues/1120

**Comment by patrickhlauke** [comment]
Would still be interested in this discussion here, as I've just had the same conversation with somebody enquiring about QR codes/cameras and whether or not that contravenes 2.5.4 (my take is still "no")
